public/static/docs/command-reference/fetch.md

```diff
@@ -74,8 +73,7 @@ by `dvc fetch` (unless the `-a` or `-T` options are used).
 
 - `-r REMOTE`, `--remote REMOTE` - name of the
```

Comment:
better to use NAME?

Reply:
Same:
```console
$ dvc fetch -h
usage: dvc fetch [-h] [-q | -v] [-j JOBS] [-r REMOTE] [-a] [-T] [-d] [-R]
                 [targets [targets ...]]
```

Reply:
Resolving as well. See Discussing in https://github.com/iterative/dvc.org/pull/1029#issuecomment-594911780 and below.